CakeMap.Rmd

```diff
@@ -16,7 +16,7 @@ different parts of Leeds, UK. The example is deliberately rather absurd to make
 it more memorable. The steps are presented in a generalisable way, to be
 applicable to a wide range of datasets.
 
-The input microdataset is a randomised ('jumbled') subset of the
+The input microdataset is a randomized ('jumbled') subset of the
```

Comment:
Oh ok, it is "F7" that tells me that it was wrong. No problem, it is the same for me.

Le Mardi 20 Janvier 2015 14:05 CET, Robin <notifications@github.com> a écrit: 
 
> > @@ -16,7 +16,7 @@ different parts of Leeds, UK. The example is deliberately rather absurd to make
> >  it more memorable. The steps are presented in a generalisable way, to be
> >  applicable to a wide range of datasets.
> >  
> > -The input microdataset is a randomised ('jumbled') subset of the
> > +The input microdataset is a randomized ('jumbled') subset of the
> 
> My version was British English, yours is American. I don't mind which we use; we should be consistent.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/Robinlovelace/spatial-microsim-book/pull/20/files#r23220345